packages/notifi-frontend-client/lib/client/clientFactory.ts

```diff
@@ -4,12 +4,16 @@ import { GraphQLClient } from 'graphql-request';
 import { NotifiFrontendConfiguration, envUrl } from '../configuration';
 import {
   NotifiFrontendStorage,
+  createInMemoryStorageDriver,
   createLocalForageStorageDriver,
 } from '../storage';
 import { NotifiFrontendClient } from './NotifiFrontendClient';
 
 export const newNotifiStorage = (config: NotifiFrontendConfiguration) => {
-  const driver = createLocalForageStorageDriver(config);
+  const driver =
+    config.storageOption === 'LocalForage'
```

Comment:
would you mind inverting this?

i.e. `(storageOption === 'InMemory') createInMemoryStorage : createLocalForageStorage`

that way it undefined values will default to the previous default

Reply:
You are right. It should be like that, thanks!